.github/workflows/cross-version-tests.yml

```diff
@@ -172,6 +172,10 @@ jobs:
           pip install --upgrade pip wheel
           pip install -e .[extras]
           pip install -r requirements/test-requirements.txt
+          if [ "${{ matrix.package }}" = "transformers" ]
+          then
+            pip install accelerate datasets
+          fi
```

Comment:
Do we need this? The requirements field for transformers contains both `accelerate` and `datasets`.

Reply:
ah, good point. Removed!